File: Finance/web/src/domain/entry/Entry.ts

export default class Entry {
    
    public id: number|null = null;
    
    public name: string|null = null;

    public amount: number|null = null;

    public userId: string|null = null;

    public entryTypeId: number|null = null;
    public entryTypeName: string|null = null;

    public dateAddedUtc: Date|null = null;

    public dateModifiedUtc: Date|null = null;
   
}


export class EntryType {

    public id: number|undefined = undefined;

    public name: string|undefined = undefined;
}

File: Finance/api/Domain/Entry/Entry.cs

namespace api.Domain.Entry;

public class Entry {

    public int? Id { get; set; }
    
    public double Amount { get; set; }
    
    public string? Name { get; set; }

    public string? UserId { get; set; }

    public int EntryTypeId { get; set; }
    
    public string EntryTypeName { get; set; }

    
    public DateTime DateAddedUtc { get; set; }

    public DateTime DateModifiedUtc { get; set; }
    
    

}